src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -388,46 +390,24 @@ class CouchObjectProvider {
      * @private
      */
     observeObjectChanges() {
-
-        let filter = {selector: {}};
-
-        if (this.openmct.objects.SYNCHRONIZED_OBJECT_TYPES.length > 1) {
-            filter.selector.$or = this.openmct.objects.SYNCHRONIZED_OBJECT_TYPES
-                .map(type => {
-                    return {
-                        'model': {
-                            type
-                        }
-                    };
-                });
-        } else {
-            filter.selector.model = {
-                type: this.openmct.objects.SYNCHRONIZED_OBJECT_TYPES[0]
-            };
-        }
-
-        // feed=continuous maintains an indefinitely open connection with a keep-alive of HEARTBEAT milliseconds until this client closes the connection
-        // style=main_only returns only the current winning revision of the document
-        let url = `${this.url}/_changes?feed=continuous&style=main_only&heartbeat=${HEARTBEAT}`;
-
-        let body = {};
-        if (filter) {
-            url = `${url}&filter=_selector`;
-            body = JSON.stringify(filter);
-        }
+        const sseChangesPath = `${this.url}/_changes`;
```

Comment:
should we worry about relative URLs for CouchDB? If so, the URL construction here won't work

Reply:
I think supporting relative paths is desirable if it's not too much extra work. What makes it fail here? Couldn't `this.url` just be a relative url?

Reply:
it can, but then would need a base variable to:
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL
> A USVString or any other object with a stringifier — including, for example, an <a> or <area> element — that represents an absolute or relative URL. If url is a relative URL, base is required, and will be used as the base URL. If url is an absolute URL, a given base will be ignored.

I can switch it back to string (and support relative URLs that way), but the query parameter stuff is really handy. It looks like the `UrlService` is doing some of this logic already (in `urlForNewTab`).